gobblin-yarn/src/main/java/gobblin/yarn/YarnService.java

```diff
@@ -94,6 +99,7 @@
   private final Config config;
 
   private final EventBus eventBus;
+  private final EventSubmitter eventSubmitter;
```

Comment:
Can you move this definition to below `private final FileSystem fs;` with an empty line in between?